INSTALL.md

```diff
@@ -118,15 +126,15 @@ be edited here.
         $ NODE_ENV="development" 
 
         ### Before you run this command, update the config/config.json with the recently created database username and password.
-        ### 
-        ### This will spew a lot of information to the screen and may
-        ### take several minutes.  It creates the tables and loads data
-        ### into UsAddress and SelectedCounties.
-        $ node data/import_into_postgres.js
 
+        ### Run sequelize migrations
```

Comment:
So the idea is, this would then update the schema created in the earlier step, to account for DB-affecting changes that happened since the SQL for that earlier step was written?

Reply:
Exactly!